src/components/Blogs/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import SearchBar from "../SearchBar";
+import { Container } from "react-bootstrap";
+import "./index.scss";
+
+export default function Blogs() {
+  const [blogs, setBlogs] = useState([]);
```

Comment:
so imagine you are me. what is blogs? all i know is blogs is an empty array. add a comment please abovei t

also we usually use "blogPosts" to refer to the posts and blog to refer to the whole site specifically.

Reply:
fixed